src/InputContainer.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import { useDispatch } from 'react-redux';
+import InputContainer from './InputContainer';
+import { ADD_RESTAURANT } from './actions';
+
+jest.mock('react-redux');
+
+describe('InputContainer', () => {
+  it('should render three inputs and one button', () => {
+    const { getByPlaceholderText, getByRole } = render(<InputContainer />);
+
+    expect(getByPlaceholderText('이름')).toBeInTheDocument();
+    expect(getByPlaceholderText('분류')).toBeInTheDocument();
+    expect(getByPlaceholderText('주소')).toBeInTheDocument();
+    expect(getByRole('button')).toBeInTheDocument();
+  });
+
+  it('restaurantInfo should be able to be entered.', () => {
+    const dispatch = jest.fn();
+    useDispatch.mockImplementationOnce(() => dispatch);
+    const { getByPlaceholderText } = render(<InputContainer />);
+
+    fireEvent.change(getByPlaceholderText('이름'), { target: { name: 'name', value: '선정릉역 깐부치킨' } });
```

Comment:
인자가 만은 경우엔 가독성을 위해 줄바꿈 처리해주는게 좋을 것 같아요~

```suggestion
    fireEvent.change(getByPlaceholderText('이름'), {
      target: {
        name: 'name',
        value: '선정릉역 깐부치킨'
      }
    });
```